common/core/desktop/src/rust/rust_mock_processor.cpp

```diff
@@ -74,6 +74,9 @@ namespace km {
       return engine_attrs;
     }
 
+    km_kbp_keyboard_key  * rust_mock_processor::get_key_list() const  {
+      return nullptr;
```

Comment:
This returns nullptr but we don't define that as a possible output for the api function `km_kbp_get_key_list()` -- shouldn't we return an empty array instead? (Same comment as mock_processor)